### ISSUE TITLE
"Cannot find module 'vscode-ipe-types'" after preparation of the development environment

### ISSUE THREAD

**Issue by MarcSkovMadsen** [lorenzo2897/vscode-ipe#168]
Hi

I would try playing around with the extension and maybe implement some improvements if possible. But after

1. I've cloned the project to my local machine
2. I've followed the [developer guide](https://github.com/lorenzo2897/vscode-ipe/blob/master/DEVELOPING.md) wrt. the section "Preparing the development environment" 

The VS Code shows me the error **Cannot find module 'vscode-ipe-types** ![Cannot find module 'vscode-ipe-types' image](https://user-images.githubusercontent.com/42288570/47251651-8933dc80-d436-11e8-9a5f-db8d6dfc4902.png).

**If possible please let me know how to solve this and update the developer guide accordingly.**

Thanks - any help is very much appreciated.

**Comment by MarcSkovMadsen** [comment]
I believe the problem is that a module called vscode-ipe-types should be installed under the node_modules folder but is not. 

I believe the contents of the vscode-ipe-types folder should be the files found in the src-common/out folder

You can see that the vscode-ipe-types module is not listed in the vscode-ipe/node_modules folder below

![image](https://user-images.githubusercontent.com/42288570/47251704-dc5a5f00-d437-11e8-9412-bbe679bd3668.png) src-common/out folder.

Instead there is a module called neuron-ipe-types 

![image](https://user-images.githubusercontent.com/42288570/47251791-2728a680-d439-11e8-8ac1-4c054f4061bc.png)

**Comment by MarcSkovMadsen** [comment]
One solution is to replace all references to 'vscode-ipe-types' with 'neuron-ipe-types'.

![image](https://user-images.githubusercontent.com/42288570/47251914-7a9bf400-d43b-11e8-9734-c05879153a17.png)

**Comment by MarcSkovMadsen** [comment]
An alternative solution would be to change the "build-types" script in the package.json file from "build-types": "tsc -p src-common/ && shx cp -r src-common/ node_modules/neuron-ipe-types/",

![image](https://user-images.githubusercontent.com/42288570/47252136-ca7cba00-d43f-11e8-92be-b8c746b43e9c.png)


to "build-types": "tsc -p src-common/ && shx cp -r src-common/ node_modules/vscode-ipe-types/",

![image](https://user-images.githubusercontent.com/42288570/47252118-783b9900-d43f-11e8-8794-cefa6b0b52b3.png)

This also solves the problem.

**Comment by MarcSkovMadsen** [comment]
I believe the first solution is the best i.e. **keeping the module name as neuron-ipe-types** as neuron is the name of this project/ add-in.

I will go ahead and create a branch for fixing the issue and create a pull request.

**Comment by LukeyQ1** [comment]
The extension was renamed from vscode-ipe to neuron-ipe (the repo will also be renamed at somepoint) however i believe all these renames were fixed. If I look at the latest version of the files they all have the correct name. Are you sure you have the latest version of the repo?

**Comment by pavanagrawal123** [comment]
I have the latest version of the repo and I'm experiencing the same issue

**Comment by LukeyQ1** [comment]
Use the 1.0.3 pull request branch

**Comment by pavanagrawal123** [comment]
Looks like the PR it was changed in: #160 was never merged.

**Comment by pavanagrawal123** [comment]
Ahh I see I was just about to comment that the PR wasn't merged. I was wondering why it wasn't merged, is the project not being worked on actively any more?

**Comment by MarcSkovMadsen** [comment]
Ok. I will use the 1.0.3 pull request branch. 

When can we expect the 1.0.3 pull request branch to be merged into master?

I would also like to know whether this is project is being maintained or not ?? Thanks :-)

**Comment by LukeyQ1** [comment]
To answer @pavanagrawal123 question are quote what I wrote in another issue thread
> I would love to work on this feature however it has been hard to find the time to work on the extension recently :(
> If anyone has a solution to this issue and would like to contribute please let me now.
> If there are any further developments I will update you

**Comment by pavanagrawal123** [comment]
@LukeyQ1 I'm considering forking the plugin if there isn't too much activity in the PRs. 

I really like this plugin and would like to fix the issues.

**Comment by LukeyQ1** [comment]
I'm currently trying to move the repo to @neuron-team and when this is done the 1.0.3 PR will be merged and it will be easier to accept pull requests such as @pavanagrawal123 . I'm currently waiting on @lorenzo2897 to move the repo.

Again I'm sorry that work has stopped on the project for now but a few of us are eager to work on it when we get the time. I can't imagine there being any other update on our end until around December at the earliest :(

**Comment by pavanagrawal123** [comment]
I'm considering a fork of this extension, as it's very cool, but it still needs a little bit of polish.

Do you have the time to merge PRs? If not I think I will be forking...

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by pavanagrawal123** [comment]
@LukeyQ1 sounds good. I have already forked to github.com/pavanagrawal123/VSNotebooks, but let's see how far the fork diverges before December!
